x-pack/plugins/observability_solution/apm/public/components/app/service_inventory/multi_signal_inventory/table/get_service_columns.tsx

```diff
@@ -91,10 +88,10 @@ export function getServiceColumns({
       sortable: true,
       width: `${unit * 9}px`,
       dataType: 'number',
-      render: (_, { environments, signalTypes }) => (
+      render: (_, { environments, dataStreamTypes }) => (
         <EnvironmentBadge
           environments={environments}
-          isMetricsSignalType={signalTypes.includes(SignalTypes.METRICS)}
+          isMetricsSignalType={dataStreamTypes.includes(EntityDataStreamType.METRICS)}
```

Comment:
nit: Not part of this PR but I think we want to check `isApmSignal(dataStreamTypes)` here

Reply:
We don't need to. The <EnvironmentBadge/> component handles it. https://github.com/cauemarcondes/kibana/blob/d4757bc3644ce28eefbe01025624439794ec8257/x-pack/plugins/observability_solution/apm/public/components/shared/not_available_popover/not_available_environment.tsx#L14